src/routes/Routes.tsx

```diff
@@ -114,6 +114,8 @@ const PriceToggleJason = React.lazy(
 
 const Pomodoro = React.lazy(() => import(/* webpackChunckName: "Pomodoro" */ '../pages/FrontendMentor/Pomodoro'))
 
+const FrontEndMentorTodo = React.lazy(() => import(/* webpackChunckName: "Todo" */ '../pages/FrontendMentor/Todo'))
```

Comment:
consider using same name for chunckname as well - ie FrontEndMentorTodo instead of Todo